StandardLibrary/runtimes/go/ImplementationFromDafny-go/Time_/externs.go

```diff
@@ -18,7 +18,7 @@ func (CompanionStruct_Default___) CurrentRelativeTime() int64 {
 	return CurrentRelativeTime()
 }
 func CurrentRelativeTime() int64 {
-	return int64(time.Now().Second())
+	return int64(time.Now().Unix())
```

Comment:
Just curious why the change?

Reply:
Second gives the time in seconds since this process started. Unix gives the epoch time. It was a bug.